sim/src/make/generator.rs

```diff
@@ -428,6 +428,56 @@ impl ScenarioGenerator {
         let mut s = Scenario::empty(map, "random people going to/from work");
         s.only_seed_buses = None;
         timer.start_iter("create people", total_ppl);
+
+        let incoming_connections = map.all_incoming_borders();
+        let outgoing_connections = map.all_outgoing_borders();
+        // transit
```

Comment:
The term "transit" in the codebase refers to buses and trains, aka, public transit. This is creating trips between two borders. I don't know a good term for this. "Pass-through trips"? Maybe just "trips between borders." It feels confusing to overload "transit" though

Reply:
"trips between borders" should be good enough, as it is a bit unusual/weird phrasing but is not presented to user